src/App.js

```diff
@@ -15,25 +15,32 @@ class App extends Component {
       popUpInfo: null,
       popUpGenre: null,
       searching: '',
+      searchPageCheck: false,
+      showAdvancedSearch: false,
```

Comment:
I moved the logic to show advanced search to App.js in the hopes of being able to get the all/card/board buttons working, but as said before, I think the solution is to have advanced search always on but hidden with 0px height when not in use